node/node.go

```diff
@@ -101,9 +102,17 @@ func (n *Node) Start(ctx context.Context) error {
 
 	// TODO(@Wondertan): Print useful information about the node:
 	//  * API address
-	//  * Pubkey/PeerID
 	//  * Host listening address
 	log.Infof("started %s Node", n.Type)
+
+	addrs, err := peer.AddrInfoToP2pAddrs(host.InfoFromHost(n.Host))
+	if err != nil {
+		log.Errorf("Error while retreiving multiaddress information: %s", err)
+		return err
+	}
+	for _, addr := range addrs {
+		log.Infow("node P2P info", "multiaddress", addr.String())
```

Comment:
Hmm, I feel like this might be simple `fmt.Println()` to differentiate it from all the logs. It is not a log of events but like something informational that is returned when you exec something in CLI. I don't have a strong opinion here, though. I

Reply:
Changed to fmt.Println as indeed it can be handy to always see at the beginning the addrs, even if you turn off logging display